### ISSUE TITLE
Teleport UI searching within connected server resource is limited

### ISSUE THREAD

**Issue by TeleLos** [gravitational/teleport#38957]
Expected behavior:
Teleport UI should allow the user to search for text within the connected terminal session. 

Current behavior:
Only text within the connected resource tab(s) is searchable. 

Bug details:
- Teleport version 15.1.1
- Chrome 122.0.6261.94 
- Recreation steps

1. Login into Teleport UI
2. Connect to Server Resource 
3. On a Mac, Issue Command + F to search for keywords within the terminal session. Only text within the connected resource tabs is searchable and not within the terminal session. 

- Debug logs

**Comment by zmb3** [comment]
Yep, looks like we need to use [`xterm-addon-search`](https://github.com/xtermjs/xterm.js/tree/master/addons/addon-search)
